src/utility/model/ball_detection.hpp

```
@@ -0,0 +1,46 @@
+
+template <typename precision_type = float>
+struct BallInferResult {
+    struct Point {
+        precision_type x{0}, y{0};
+    };
+
+    Point point{};
+
+    precision_type confidence{0};
+
+    auto unsafe_from(std::span<const precision_type> raw) noexcept -> void {
```

Clang (14.0.6)

[warning] parameter 'raw' is unused

(misc-unused-parameters)